src/test/resources/org/broadinstitute/hellbender/tools/walkers/ValidateVariants/gvcf.basepairResolution.vcf

```diff
@@ -0,0 +1,10078 @@
+##fileformat=VCFv4.1
+##ALT=<ID=NON_REF,Description="Represents any possible alternative allele at this location">
```

Comment:
This is a pretty big file.  Could you shrink it to fewer lines and still have a useful test or is the length important?